src/index.ts

```diff
@@ -1,14 +1,17 @@
+import {AEGTree} from "./AEG/AEGTree";
+import {CutNode} from "./AEG/CutNode";
+import {Ellipse} from "./AEG/Ellipse";
+import {AtomNode} from "./AEG/AtomNode";
+import {ellipseCreation, removeCutListener} from "./EllipseCreation";
+import {atomCreation, removeAtomListener} from "./AtomCreation";
+
 /**
  * A program to draw ellipses and atoms.
  * @author Dawn Moore
  * @author James Oswald
  */
 
-import {AEGTree} from "./AEG/AEGTree";
-import {Point} from "./AEG/Point";
-import {CutNode} from "./AEG/CutNode";
-import {Ellipse} from "./AEG/Ellipse";
-import {AtomNode} from "./AEG/AtomNode";
+import "./index.css";
```

Comment:
This CSS shouldn't be necessary anymore. I got rid of this when we swapped to Vite, which can actually detect the includes from the HTML.
```suggestion

```

Reply:
Yeah I thought so too, but when I took it out this is what the HTML turned into

![d60cd47022f5a3ecf4110e10a043aff1](https://github.com/RAIRLab/Peirce-My-Heart/assets/93628226/3c7daeeb-6429-4dca-93ae-7fceb8bddc62)

Reply:
Resolved via `npx vite preview` / `npm run live-edit`